yarn.lock

```diff
@@ -2831,6 +2831,14 @@ better-sqlite3@^8.2.0:
     bindings "^1.5.0"
     prebuild-install "^7.1.0"
 
+better-sqlite3@^8.4.0:
```

Comment:
isn't better-sqlite3 already in our yarn.lock? Are we introducing another version of it or something?